components/chat_ui/common/chat_ui_utils.h

```diff
@@ -0,0 +1,16 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_CHAT_UI_COMMON_CHAT_UI_UTILS_H_
+#define BRAVE_COMPONENTS_CHAT_UI_COMMON_CHAT_UI_UTILS_H_
+
+#include "build/build_config.h"
+
+namespace chat_ui {
```

Comment:
nit: for consistency, new line here too

```suggestion
namespace chat_ui {

```